app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -121,30 +146,7 @@
       android:layout_width="match_parent"
       android:layout_height="match_parent" />
 
```

Comment:
In the current implementation, the `hints_and_solution_fragment_placeholder` frameLayout is of no use. So, you can delete this layout and after that, you would not require to shift `hints_and_solution_fragment_container` above.

Reply:
Even after removing the "hints_and_solution_fragment_placeholder" i would need to add the "hints_and_solution_fragment_container" inside the constraint layout as I'm adding a constraint from progressbar to  "hints_and_solution_fragment_container"

Reply:
@aishapervin Definitely, you are right but we want to get rid of `hints_and_solution_fragment_placeholder`, so it needs to get deleted and yes you'll have to shift it up in the constraint layout.

Reply:
done.